scv-prototype-frontend/src/app/views/Private/Profile/DatesOfBirth/DatesOfBirth.jsx

```diff
@@ -29,11 +35,13 @@ const DatesOfBirth = ({ datesOfBirth }) => {
 };
 
 DatesOfBirth.propTypes = {
-  datesOfBirth: PropTypes.arrayOf(PropTypes.shape(DateOfBirth.propTypes))
+  person: PropTypes.objectOf(PropTypes.object),
+  programs: PropTypes.arrayOf(PropTypes.object)
```

Comment:
It would be better to use `PropTypes.shape` here.

Reply:
Fixed in next commit.